### DISCUSSION TITLE
API reference

### DISCUSSION THREAD

**Discussion by bronf** [requarks/wiki#6086] [Help / Questions] [answered]
Hi,
Where can I find the reference of the API? I would like to have the list of everything I can query.
In particular, I want to update the groups using the API, i.e. add/remove users.
Thanks

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/dev/api

**Reply by bronf** [reply]
I've seen that page but it does not look like a reference. I mean there are examples but it is all what the API can do?

**Reply by NGPixel** [reply]
As stated in the very first section on that page:

> The GraphQL endpoint is located at `/graphql` on your wiki.js site.
> 
> You can access this endpoint from your browser to load the GraphQL Playground tool which lets you build and test queries, as well as > explore all the possible resources you can access. **The docs for all available queries and mutations are accessible on the right side of the screen.**

**Reply by bronf** [reply]
Awesome, I had been to the playground but did not see the doc on the right hand side. I'm so sorry to waste your time and thank you for your kindness.
